kolibri/core/errorreports/models.py

```diff
@@ -47,25 +54,55 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):
 
 
 class ErrorReports(models.Model):
```

Comment:
Also noting that https://github.com/learningequality/kolibri/blob/24186194c23f2f0dd126e72bdc79ed4b5ddd413f/kolibri/core/device/api.py#L201 is a very important piece of information to store.

Reply:
Similarly, we should store `is_touch_device ` to clearly distinguish between touch and non touch devices